src/Microsoft.AspNet.Authentication.OAuth/OAuthAuthenticationOptions.cs

```diff
@@ -77,7 +69,7 @@ public string Caption
         /// <value>
         /// The back channel timeout.
         /// </value>
-        public TimeSpan BackchannelTimeout { get; set; }
+        public TimeSpan BackchannelTimeout { get; set; } = TimeSpan.FromSeconds(60);
```

Comment:
FxCop typically warns about setting virtual properties in ctors, and we don't fully control the derived classes so...

http://stackoverflow.com/questions/465436/is-it-ok-to-call-virtual-properties-from-the-constructor-of-a-nhibernate-entity